frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -369,14 +372,42 @@ const SECTIONS = updateSectionsWithPlugins({
         getHidden: (_, derivedSettings) => !derivedSettings["enable-embedding"],
       },
       {
-        widget: EmbeddingCustomizationInfo,
-        getHidden: (_, derivedSettings) =>
-          !derivedSettings["enable-embedding"] ||
-          MetabaseSettings.isEnterprise(),
+        widget: EmbeddingOption,
+        getHidden: (_, derivedSettings) => !derivedSettings["enable-embedding"],
+        embedName: t`Standalone embeds`,
+        embedDescription: t`Securely embed individual questions and dashboards within other applications.`,
+        embedType: "standalone",
+      },
```

Comment:
Use a similar approach to `AuthenticationOption` e.g. 
https://github.com/metabase/metabase/blob/dfe4475a77fc4a910a52230ac76a0899bfd7c646/frontend/src/metabase/plugins/builtin/auth/google.js#L30-L36